### PR TITLE
add links

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: website/src/theme/Footer/index.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ function Footer() {
           <section className={styles.footerLinks}>
             {/* <a href="https://platform.icgc-argo.org/contact">Contact</a>
             <Slash /> */}
-            <a href="https://platform.icgc-argo.org/privacy">Privacy Policy</a>
+            <a href="https://www.icgc-argo.org/page/2/privacy">Privacy Policy</a>
```
Comment (hlminh2000) [inline website/src/theme/Footer/index.js:38]: these urls look unstable, should we provide them in env instead? (doesn't have to be now, but if it's expected to change, it seems clunky to have to wait for a software release)

Comment (ciaranschutte) [reply website/src/theme/Footer/index.js:38]: Definitely a bit clunky but is it better to have several env vars just for links?
These aren't clean urls (/page/2/, /page/77/) so I'm wary of even having a base url as an env var :(

**Comment by ciaranschutte** [comment]
@rosibaj will change these to env variables. blocked by moving to Netlify ticket....looking for ticket now...

**Comment by rosibaj** [comment]
- afik not expected to change
- isnt it a quick deploy to update them? not sure this needs to be blocked by env variables or netlify

**Comment by ciaranschutte** [comment]
Links could easily change. For example if they change any kind of permalink setting in WP (which I think they're using?) Regardless they're generated links and could change.

**Comment by joneubank** [comment]
Is it easier to change the value of the link in the Footer code or in an environment variable? With how straightforward our deploy process will be it might be fine to leave these hardcoded... as easy to update as changing any other content.

**Pull request merged**